roles/etcd_migrate/tasks/migrate.yml

```diff
@@ -42,6 +44,7 @@
     --lease-duration 1h
   environment:
     ETCDCTL_API: 3
+    PATH: "/usr/local/bin:/var/usrlocal/bin:{{ ansible_env.PATH }}"
```

Comment:
Oh I see what you mean. I did do a quick dependency check and the role does not depend on ``openshift_facts`` however the playbook does.

@ingvagabund would it be safe to use ``openshift.common.admin_binary`` in the role?